.github/test_dependencies.yaml

```
@@ -18,15 +18,19 @@ e2e:
   # For Istio, we define combinations of Kind and Istio versions that will be
   # used directly in the test matrix `include` section.
   istio:
+      # renovate: datasource=docker depName=kindest/node versioning=docker
     - kind: 'v1.28.0'
       # renovate: datasource=docker depName=istio/istioctl versioning=docker
       istio: 'v1.19.0'
```

Wouldn't it be better like so:

```suggestion
    - # renovate: datasource=docker depName=kindest/node versioning=docker
      kind: 'v1.28.0'
      # renovate: datasource=docker depName=istio/istioctl versioning=docker
      istio: 'v1.19.0'
```

?

---

Yeah, makes sense. 👍  PTAL